README.md

```diff
@@ -356,6 +358,96 @@ reached. If no configuration file is found, Regal will use the default configura
 A custom configuration may be also be provided using the `--config-file`/`-c` option for `regal lint`, which when
 provided will be used to override the default configuration.
 
+## Ignoring Rules
+
+If some rule doesn't align with your team's preferences, don't worry! Regal is not meant to be the law, and some rules
+may not make sense for your project, or parts of it. Regal provides several different ways to ignore rules, either
+entirely, or with more granularity.
+
+### Ignoring a Rule Entirely
+
+If you want to ignore a rule entirely, set its level to `ignore` in the configuration file:
+
+```yaml
+rules:
+  style:
+    prefer-snake-case:
+      # Use camel case to comply with our naming conventions
```

Comment:
idea: I might use an example company name here, e.g.

```suggestion
      # At example.com, we use camel case to comply with our naming conventions
```

I feel this might make 'our' clearer, but feel free to ignore this.

Reply:
👍